arch/arm/src/stm32/Kconfig

```diff
@@ -2027,6 +2026,7 @@ config STM32_STM32G43XX
 config STM32_STM32G431K
 	bool
 	default n
+	select STM32_HAVE_DAC3
```

Comment:
Could you move this under STM32_STM32G43XX option ?

Reply:
Yes, I will move the option as you suggested.

Thanks